test/test_helper.rb

```diff
@@ -42,12 +42,14 @@ def create_default_stubs
     SseRailsEngine.stubs(:send_event).returns(true)
     Project.any_instance.stubs(:clone_repository).returns(true)
     Project.any_instance.stubs(:clean_repository).returns(true)
+    DateTime.stubs(:now).returns(DateTime.new(2001,2,3,4,5,6))
```

Comment:
not a fan of this ... prefer doing it in the affected tests ... maybe add a global `freeze_time '2001-02-01 03:04:05'` kind of helper